x-pack/plugins/aiops/public/components/log_rate_analysis/log_rate_analysis_content/log_rate_analysis_attachments_menu.tsx

```diff
@@ -88,6 +107,37 @@ export const LogRateAnalysisAttachmentsMenu = () => {
                 },
               ]
             : []),
+          ...(canUpdateCase || canCreateCase
+            ? [
+                {
+                  name: i18n.translate('xpack.aiops.logRateAnalysis.attachToCaseLabel', {
+                    defaultMessage: 'Add to case',
+                  }),
+                  'data-test-subj': 'aiopsLogRateAnalysisAttachToCaseButton',
+                  disabled: !hasSignificantItemsToAttach,
+                  ...(!hasSignificantItemsToAttach
+                    ? {
+                        toolTipProps: { position: 'left' as const },
+                        toolTipContent: i18n.translate(
+                          'xpack.aiops.logRateAnalysis.attachToCaseTooltipContent',
+                          {
+                            defaultMessage:
+                              'Cannot add to case because the analysis did not produce any results',
```

Comment:
If I hit 'Reset analysis', the 'Add to case' action is enabled. Can we disable the action after hitting reset?

<img width="1326" alt="Screenshot 2024-11-27 at 11 13 29" src="https://github.com/user-attachments/assets/79853059-58ba-4c34-b97e-e8355f81f397">

Reply:
Is it possible to have a different message here depending on the two use cases:

1. Analysis did not produce any results
2. The user has yet to run an analysis

Reply:
Done in:
[#99cca0b](https://github.com/elastic/kibana/pull/201549/commits/99cca0b1bf263abead23374f59945559d1f46155)